src/runtime-rs/fuzz/README.md

```diff
@@ -0,0 +1,25 @@
+# Fuzz testing
+To get started with fuzz testing you'll need to install cargo-fuzz.
+
+`cargo install cargo-fuzz`
+
+You can list the available fuzz harnesses using cargo fuzz. e.g.
+```
+$ cd src/runtime-rs/fuzz
+$ cargo +nightly fuzz list
+fuzz_hypervisor_device_roundtrip
+```
+
+To run a fuzz harness simply use the following command.
+
+```
+$ cargo +nightly run fuzz_hypervisor_device_roundtrip
```

Comment:
The same question as the previous one.

Reply:
See response above. Nightly is required for instrumentation. But no nightly API's are used to write the fuzzer's themselves.